tests/flat-config.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-env mocha */
+
+'use strict';
+
+const semver = require('semver');
+const eslintPkg = require('eslint/package.json');
+
+if (!semver.satisfies(eslintPkg.version, '>= 8.57.0')) {
+  return;
+}
```

Comment:
flat config wasn't added in 8.57; is there a reason this isn't targeting the first version of 8 that has flat config?

Reply:
cool, i can do that.

Reply:
Here's also a commit that updates this: https://github.com/mdjermanovic/eslint-plugin-react/commit/2f28188a12e4f2657276641b842cd2d9f01fe744